package/Object_storage/object_storage.py

```diff
@@ -0,0 +1,66 @@
+from package.object_manager.manager import ObjectManager
+
+class ObjectStorage:
+    def __init__(self):
+        self.objManager = ObjectManager()
+        self.storage = dict()
+        self.SWO = SWO()
+        self.PW = PW()
+        self.counter = 0
+
+    def delete(self, uid):
+        del self.storage[uid]
+
+    def update(self,uid, new_data:dict):
```

Comment:
you also need to update obj in object manager. And redraw mesh in PW. Update SWO